notebooks/official/generative_ai/nvidianims_vertexai.ipynb

```diff
@@ -0,0 +1,729 @@
+{
```

Comment:
<p>Use "[your-project-id]" for PROJECT_ID</p><p>Suggest to set a default ZONE/LOCATION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Resolved.